InternxtDesktop/Views/Backup/DeleteBackupDialog.swift

```diff
@@ -48,6 +48,7 @@ struct DeleteBackupDialog: View {
         .background(colorScheme == .dark ? Color.Gray1 : Color.white)
         .cornerRadius(10)
         .frame(width: 320)
+        .shadow(color: .black.opacity(0.1), radius: 1, x: 0, y: 1)
```

Comment:
this shadow is not correct, check the Figma properties

![image](https://github.com/internxt/drive-desktop-macos/assets/42890253/831cb3d7-89cb-4f66-bc5a-1e86732bfb06)